### PR TITLE
Dynamodb inline

### PR DESCRIPTION
**Pull request by svenngjeto**
Definition of nva_publications is inline in template, table and index names as parameters with default values

### CODE REVIEW

**Review by axthosarouris — APPROVED** [review]
Approved on the condition that !Ref PublicationsTable returns the name and not the ARN of the table.

File: template.yaml [hunk 1]
```diff
@@ -299,14 +299,14 @@ Resources:
       Environment:
         Variables:
           ALLOWED_ORIGIN: '*'
-          TABLE_NAME: !Ref PublicationsTableName
-          BY_PUBLISHER_INDEX_NAME: !Ref PublicationsByOwnerIndexName
+          TABLE_NAME: !Ref PublicationsTable
+          BY_PUBLISHER_INDEX_NAME: !Ref PublicationsTable
```
Comment (norrs) [inline template.yaml:303]: Intended?

Comment (svenngjeto) [reply template.yaml:303]: Nope..

File: template.yaml [hunk 2]
```diff
@@ -66,7 +66,7 @@ Resources:
       DefinitionBody:
         swagger: '2.0'
         info:
-          title: NVA Publication API
+          title: NVA custom Publication API
```
Comment (norrs) [inline template.yaml:69]: Probably dont need the `custom` here either @svenng ?

**Review by norrs — APPROVED** [review]
nitpick, looks good.

File: template.yaml [hunk 3]
```diff
@@ -299,14 +299,14 @@ Resources:
       Environment:
         Variables:
           ALLOWED_ORIGIN: '*'
-          TABLE_NAME: !Ref PublicationsTableName
+          TABLE_NAME: !Ref PublicationsTable
```
Comment (janbj) [inline template.yaml:302]: Maybe we should make the table name and index names internal to simplify stack setup?

EDIT: I guess it helps with default values.

**Review by janbj — APPROVED** [review]

File: template.yaml (hunk 3 above)
Comment (svenngjeto) [reply template.yaml:302]: yes, using defaults without overriding them is poor man's constants

**Comment by norrs** [comment]
@svenngjeto : Should we maybe see if we can get this merged? :)

**Pull request closed without merging**